ports/litex/mpconfigport.mk

```diff
@@ -18,6 +18,7 @@ CIRCUITPY_AUDIOIO = 0
 CIRCUITPY_BITBANGIO = 0
 CIRCUITPY_BOARD = 0
 CIRCUITPY_BUSIO = 0
+CIRCUITPY_SDCARDIO = 0
```

Comment:
This is a complete nit, could you put the SDCARDIO one in alphabetcial order with the others?